src/components/layers/VectorLayer.vue

```diff
@@ -274,8 +274,14 @@ function polygonCut(polygon, line, properties) {
       }
     }
 
+    properties['id'] = new Date().getTime()
     cutPolyGeoms.forEach(function(geometry) {
-      cutFeatures.push(turf.polygon(geometry, properties));
+      if (Object.keys(properties).includes('id')){
+        properties['id'] = properties['id'] + 1
```

Comment:
+ +1 doesn't seem to make it unique, maybe better to call 'randId()' to generate a suffix.

Reply:
randId() is used to generate the ID.